doc/iterations/2024.11/add_support_for_form_property_section_layout.adoc

```diff
@@ -0,0 +1,35 @@
+= Add support for form property section layout
+
+== Problem
+
+A property section in a form is made of a label, a help icon and a edition area.
+Currently, the layout is always vertical: label and help icon are in div and the edition area is below.
+
+== Key Result
+
+It will be possible as a specifier to choose if the layout is horizontal or vertical.
+
+== Solution
+
+A new attribute `flexDirection` is added on form widget styles. The value is among `[row, rowReverse, column, columnReverse]`.
```

Comment:
One line per sentence

Reply:
For the existing exemple CheckboxPropertySection, we didn't use the same naming
`labelPlacement: 'end' | 'start' | 'top' | 'bottom';`